### PR TITLE
feat--added scroll to top button

### PR DESCRIPTION
**Pull request by Arnabhit**
Added the scroll to top button for the issue #18. Please review this. Thank you.

### CODE REVIEW

**Comment by ArnabChatterjee20k** [comment]
add a video for showing demonstartion

**Comment by Arnabhit** [comment]
https://github.com/DevSoc-Asansol2023/DevSoc-Main/assets/113552756/fe831109-c86c-46f7-a8ce-f0215c5166c6

**Comment by Arnabhit** [comment]
[Screen Recording - Made with RecordCast.webm](https://github.com/DevSoc-Asansol2023/DevSoc-Main/assets/113552756/c056dcc3-dcb5-4ae8-a377-52bbb543cd91)

**Comment by ArnabChatterjee20k** [comment]
Mobile view?

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.10.1",
     "react-intersection-observer": "^9.5.2",
-    "react-spring": "^9.7.2"
```
Comment (ArnabChatterjee20k) [inline package.json:19]: where you have used react-spring and styled components?

**Review by ArnabChatterjee20k — COMMENTED** [review]
Where you have used react spring and styled components

File: package.json (hunk 1 above)
Comment (Arnabhit) [reply package.json:19]: Mobile View
[Screen Recording - Made with RecordCast (1).webm](https://github.com/DevSoc-Asansol2023/DevSoc-Main/assets/113552756/d56a17de-dc20-4540-af6b-7029f0b56f96)

**Comment by Arnabhit** [comment]
[Screen Recording - Made with RecordCast (1).webm](https://github.com/DevSoc-Asansol2023/DevSoc-Main/assets/113552756/7f838e03-e629-4419-85e8-73e8d1c23f14)


Mobile View

File: package.json (hunk 1 above)
Comment (Arnabhit) [reply package.json:19]: I have used useEffect and usestate hooks to do it. Is there any problem

**Comment by ArnabChatterjee20k** [comment]
Its looking and working nice
But please take care of that package thing

File: package.json (hunk 1 above)
Comment (ArnabChatterjee20k) [reply package.json:19]: No its ok
Actually it was showing multiple versions of react spring
my bad I am merging it

**Review by ArnabChatterjee20k — APPROVED** [review]

**Pull request merged**